src/Core/EMS.Application/Common/Constants/ResponseMessageConstant.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace EMS.Application.Common.Constants
+{
+    public class ResponseMessageConstant
+    {
+        public const string EMPLOYEE_SAVE_SUCCESS_MESSAGE = "Employee has been saved successfully.";
+        public const string EMPLOYEE_UPDATE_SUCCESS_MESSAGE = "Employee has been updated successfully.";
+        public const string EMPLOYEE_GETBYID_FAIL_MESSAGE = "Employee can not found.";
+        public const string EMPLOYEE_GETBYID_SUCCESS_MESSAGE = "Employee found.";
+        public const string EMPLOYEEDEACTIVATE_FAIL_MESSAGE = "Employee already deActive";
+        public const string EMPLOYEEDEACTIVATE_SUCCESS_MESSAGE = "Employee deActivated successfully.";
```

Comment:
Please change the variable name 'EMPLOYEEDEACTIVATE_SUCCESS_MESSAGE' to adhere to proper naming conventions.